docs/quickstart.md

```diff
@@ -0,0 +1,64 @@
+# FastTrack Quickstart
+
+## Install Dependencies
+
+FastTrack requires the following dependencies to be installed on your system:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Docker Buildx](https://docs.docker.com/buildx/working-with-buildx/)
+- [Python 3](https://www.python.org/downloads/)
+- [MLFlow](https://mlflow.org/docs/latest/index.html)
+
+## Build FastTrack
+
+FastTrack can be built using the following command:
+
+```bash
+# Install json parser
+sudo apt-get install jq
+
+# Get the build tags and version from the settings.json file
+tags="$(jq -r '."go.buildTags"' .vscode/settings.json)"
+version=$(git describe --tags | sed 's/^v//')
+
+docker build --build-arg tags="$tags" --build-arg version="$version" -t fasttrack .
+```
+
+## Run FastTrack
+
+FastTrack can be run using the following command:
+
+```bash
+docker run --rm -p 5000:5000 -ti fasttrack
+```
+
+Verify that you can see the UI by navigating to http://localhost:5000/.
+
+![FastTrack UI](https://files.mcaq.me/57b05.jpg)
+
```

Comment:
we probably just want to show how to _run_ fasttrack in the quickstart, which can be done with the DockerHub image (`gresearch/fasttrack`)

Reply:
Yeah, good point. I'll remove the build steps and make that a seperate doc :))